docs/apache-airflow/core-concepts/params.rst

```diff
@@ -309,6 +313,10 @@ The following features are supported in the Trigger UI Form:
   If you want to change values manually, the JSON configuration can be adjusted. Changes are overridden when form fields change.
 - If you want to render custom HTML as form on top of the provided features, you can use the ``custom_html_form`` attribute.
 
+.. note::
+    If the field is required you need to specify a valid default value as well. Exception is if DAG is defined with
```

Comment:
what do you mean "_valid_ default"

Reply:
Sorry, the not native speaker english... thanks for the feedback. Tried to re-phrase a bit - better now?